### PR TITLE
docs: Update readme

### PR DESCRIPTION
**Pull request by prm-kevingrandon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +43,29 @@ export default {
...
 ```
 
+### Parent Component
+
+```vue
+<script setup lang="ts">
+import { createEventLogger } from 'promoted-snowplow-logger';
+import { v4 } from "uuid";
+
+const handleError = process.env.NODE_ENV !== 'production' ? (err) => { throw err; } : console.error;
```
Comment (prm-dan) [inline README.md:53]: Snit - Yuu can also link to our integration documentation.

https://docs.promoted.ai/docs/how-to-your-web-events-with-promoted-metrics

File: README.md [hunk 2]
```diff
@@ -30,6 +43,29 @@ export default {
...
+  enabled: true,
+  platformName: 'mymarket',
+  handleError,
+}).logImpression;
+
+const uuid = v4;
+</script>
+
+<template>
+  <YourComponent :logImpression="logImpression" :uuid="uuid" :handleError="handleError">
```
Comment (prm-dan) [inline README.md:65]: Might be good to code up a simple click handler example too.

**Review by prm-dan — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (prm-kevingrandon) [reply README.md:53]: Adding a link to this doc.

File: README.md (hunk 2 above)
Comment (prm-kevingrandon) [reply README.md:67]: Might need some changes for this, I'll do this in a follow-up.

**Pull request merged**